package.json

```diff
@@ -48,7 +48,9 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "regenerator-runtime": "0.13.11",
+    "rollbar": "2.26.4",
     "rxjs": "7.8.1",
+    "sanitize-html": "2.13.0",
     "tslib": "2.4.1",
```

Comment:
Is Agora compatible with the latest version (2.7.0)?

Reply:
I think 2.13.0 is the latest according to:  [link](https://www.npmjs.com/package/sanitize-html)

Reply:
You are right.